cmd/kops/root.go

```diff
@@ -148,9 +146,13 @@ func initConfig() {
 		viper.SetConfigFile(rootCommand.configFile)
 	}
 
+	viper.SetEnvPrefix("KOPS")
+	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 	viper.SetConfigName(".kops") // name of config file (without extension)
-	viper.AddConfigPath("$HOME") // adding home directory as first search path
-	viper.AutomaticEnv()         // read in environment variables that match
+	viper.AddConfigPath(".")     // use current directory first
+	viper.AddConfigPath("$HOME")
+	viper.AddConfigPath("$HOME/.config/kops") // standard linux config dir
```

Comment:
Yes, I agree it would be great to switch this to ~/.kops/config.  It looks like we can _add_ an alternative location of `~/.kops/config`, which would avoid breakage also